Js/test.js

```diff
@@ -0,0 +1,23 @@
+const searchByName = require('./main.js');
+const data =require('./data.js')
+test('should return all the objects contain the pass name',()=>{
+    const actual=searchByName('Formal')
```

Comment:
you want to test search by name function  but you pass 'formal' which is a category of an item, not the name